utils/hct/hctdb.py

```diff
@@ -5508,13 +5511,43 @@ def UFI(name, **mappings):
         )
         next_op_idx += 1
 
-        # End of DXIL 1.8 opcodes.
-        self.set_op_count_for_version(1, 8, next_op_idx)
+        self.set_op_count_for_version(1, 9, next_op_idx)
```

Comment:
There should be no change to these lines at all.  This is the end-of-DXIL 1.8 marker which follows the same pattern as prior DXIL versions.  After DXIL 1.8, you add new opcodes below this without changing the marker.